app/models/embedded_ansible_worker/runner.rb

```diff
@@ -40,11 +39,10 @@ def setup_ansible
   end
 
   def do_work
-    if EmbeddedAnsible.running?
-      _log.info("#{log_prefix} supervisord is ok!")
+    if EmbeddedAnsible.alive?
+      heartbeat
```

Comment:
@Fryguy yeah, we'll keep trying to fix the situation by reconfiguring and starting the service via the setup script until the worker hasn't responded.  It was the simplest thing to do until we figure out what ways the service can fail.